common-content/en/module/tools/virtual-environments/index.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "Virtual environments"
+headless = true
+time = 30
+facilitation = false
+emoji= "📖"
+objectives=[
+    "Create a virtual environment with some dependencies installed.",
+]
++++
+
+We often need to use libraries in Python. 
+
+Python handles dependencies differently from JavaScript, but it has similarities.
+
+We've seen that in JavaScript we write down what dependencies we need in a `package.json` file, and when we run `npm install` they will get fetched into a folder called `node_modules`.
+
+In Python, we write down what dependencies we need in a file called `requirements.txt`. It doesn't contain JSON, it just contains a list of dependencies, one per line.
+
+### Virtual environments
+
+To install the dependencies, we need to make something called a virtual environment, where they will get installed to.
```

Comment:
Define virtual environment. Use a simple concrete analogy and pop it in a blockquote.

Reply:
Rather than an analogy, I did a comparison with `node_modules`, which even let me discuss configuration vs convention :)